### DISCUSSION TITLE
Feature request: Custom episode lists

### DISCUSSION THREAD

**Discussion by Solomon1732** [amugofjava/anytime_podcast_player#73] [Ideas]
Hello. Is it possible to add the ability to create custom episode lists? I find it to be incredibly convenient. It would enable users (such as myself) to create lists such as "to listen", "saved", "subject XYZ", and so on. I saw that one of the features in your roadmap is a queue. I believe it might be possible to extend the queue to what I ask for.

I very recently switched to your app, and so far I've been liking it 😄 Thanks for developing it!

**Comment by amugofjava** [comment]
Hi @Solomon1732,

It's great to hear you are liking Anytime.

The queue function is currently being tested and I hope to roll this out soon. I would like to support custom lists/playlists, but this is on the longer-term roadmap. Beforehand, I would like to make it easier to find episodes and plan to add an episode view. This will display a list of episodes across your library which would be filterable: All, Started but not finished, most recent, favourites etc. This would then lead on adding these episodes to a custom list for easy playback.

Thank you for your suggestion. If you have any further ideas or suggestions I would love to hear them.

**Reply by Solomon1732** [reply]
Now I understand. Thanks 😃
